README.md

```diff
@@ -70,8 +70,26 @@ conda install -c pyronear pyrovision
 You can use the library like any other python package to detect wildfires as follows:
 
 ```python
-from pyrovision.datasets import OpenFire
-dataset = OpenFire('./data', download=True)
+from pyrovision.models.rexnet import rexnet1_0x
+from torchvision import transforms
+import torch
+from PIL import Image
+
+
+# Init
+normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
+
+tf = transforms.Compose([transforms.Resize(size=(448)), transforms.CenterCrop(size=448),
+                         transforms.ToTensor(), normalize])
+
+model = rexnet1_0x(pretrained=True).eval()
+
+# Predict
+im = Image.open("path/to/your/image.jpg").convert('RGB')
+imT = tf(im)
+
+with torch.no_grad():
+    is_wildfire = torch.sigmoid(pred).item() >= 0.5
```

Comment:
I think you're missing `pred = model(tf(im).unsqueeze(0))` over here :)

Reply:
oups :)